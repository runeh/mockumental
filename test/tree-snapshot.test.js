const { resolve } = require('path');

const { load } = require('../lib/dirparser');

const fixtureDir = resolve(__dirname, 'fixture-tree');

it('dir parsing snapshot smoke test', () => {
    const tree = load(fixtureDir);
    expect(tree).toMatchSnapshot();
});
