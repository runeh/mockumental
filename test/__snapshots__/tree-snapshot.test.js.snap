// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dir parsing snapshot smoke test 1`] = `
Array [
  Object {
    "handlers": Array [
      Object {
        "current": true,
        "handlerId": "r1h1",
        "handlerType": "static",
        "isDefaultHandler": false,
        "method": "GET",
        "mimeType": "application/json",
        "name": "maintenance.200.json",
        "path": "/Users/runeh/Dropbox/prosjekter/mockumental/test/fixture-tree/maintenance.200.json",
        "prettyName": "maintenance",
        "proxyUrl": null,
        "routeId": "r1",
        "status": 200,
        "type": "json",
      },
      Object {
        "current": false,
        "handlerId": "r1h2",
        "handlerType": "static",
        "isDefaultHandler": false,
        "method": "GET",
        "mimeType": "application/json",
        "name": "ok.json",
        "path": "/Users/runeh/Dropbox/prosjekter/mockumental/test/fixture-tree/ok.json",
        "prettyName": "ok",
        "proxyUrl": null,
        "routeId": "r1",
        "status": 200,
        "type": "json",
      },
    ],
    "method": "GET",
    "path": "/",
    "routeId": "r1",
  },
]
`;
